code/modules/power/tesla/energy_ball.dm

```diff
@@ -190,7 +166,7 @@
 	*/
 	var/atom/closest_atom
 	var/closest_type = 0
-	var/static/things_to_shock = typecacheof(list(/obj/machinery, /mob/living, /obj/structure, /obj/vehicle/ridden))
+	var/static/things_to_shock = typecacheof(list(/obj/machinery, /turf/closed/wall, /obj/structure, /obj/vehicle/ridden))
```

Comment:
This makes it remarkably safer to approach tesla anomalies, and to go through electrical storms that don't explode, since they can't hurt people
Also I don't think there's any behavior for shocking walls, in testing arcs don't seem to target them even if there's nothing else nearby